StPetersburg/InEtn40556.xml

```diff
@@ -0,0 +1,223 @@
+                          <extent>
+                              <measure unit="leaf">3</measure>
+                              <dimensions type="outer" unit="mm">
+                                <height>2080</height>
+                                <width>70</width>
+                              </dimensions>
+                          </extent>
+
+                      </supportDesc>
+                      <layoutDesc>
+                          <layout columns="1" writtenLines="300">
+                          </layout>
+                      </layoutDesc>
+                  </objectDesc>
+                  <handDesc>
+                    <handNote script="Ethiopic" xml:id="h1">
+                        <seg type="ink">Black, red</seg>
+                        <seg type="script">The script is archaic, angular according to <bibl><ptr target="bm:Chernetsov1975Svitki"/>
+                        <citedRange unit="page">221</citedRange></bibl>.</seg>
+                        <date><date>Second half of the 18th-cent - beginning of the 19th century</date> according to <bibl><ptr target="bm:Chernetsov1975Svitki"/>
+                        <citedRange unit="page">221</citedRange></bibl>.</date>
+                    </handNote>
+                  </handDesc>
+                  <decoDesc>
+                    <decoNote xml:id="d1" type="drawing">
+                        <desc>Drawing in black, red, and yellow, of an eight-pointed <term key="cross">cross</term> with a pedestal and two
+                      <term key="angel">angels</term> on both sides. The drawing resembles that of
+                      <ref corresp="InEtn210322" type="mss"></ref>.</desc>
+                    </decoNote>
+                    <decoNote xml:id="d2">
+                      <desc>Drawing in black, red, and yellow, of an <term key="angel">angel</term>
+                      with a <term key="cross"></term> on the head,
+                      with a <term key="sword">sword</term>.</desc>
+                    </decoNote>
+                    <decoNote xml:id="d3">
+                      <desc>Drawing in black, red, and yellow, of a "bound devil" - devil's face with eight horns in a frame of four
+                        eight-pointed crosses according to <bibl><ptr target="bm:Chernetsov1975Svitki"/>
+                      <citedRange unit="page">221</citedRange></bibl>. The drawing resembles the
+                      <ref type="authFile" corresp="AT1140SealSol"></ref> and has been published in <bibl><ptr target="bm:Chernetsov1975Svitki"/>
+                      <citedRange unit="page">221</citedRange></bibl>.</desc>
+                    </decoNote>
+
+                  </decoDesc>
+                  <additions>
+                    <list>
+                      <item xml:id="e1">
+                        <desc>
+                          A paper shit with a description of scroll by an unknown cataloguer has been glued to the reverse
```

Comment:
sheet :)
description of the scroll?

Reply:
I will correct that as soon as I can fetch the origin :)